### PR TITLE
[e2e] add e2e tests

### PR DESCRIPTION
**Pull request by pducolin**
Meant to run locally, for now

They implement in go https://github.com/DataDog/experimental/blob/main/teams/agent-platform/package-scripts-test/install_test.sh

### CODE REVIEW

File: test/e2e/agentplatform.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+	if len(*af) > 0 {
+		return fmt.Errorf("flavor flag already set to %s while trying to set to %s", *af, value)
+	}
+	fmt.Printf("Setting flavor flag to %s\n", value)
+	*af = agentFlavor(value)
+	return nil
+}
+
+const (
+	scriptURL    = "https://dd-agent.s3.amazonaws.com/custom"
```
Comment (chouetz) [inline test/e2e/agentplatform.go:35]: It means we might state somewhere we need to push the install_scripts to be tested in the `custom` folder prior launching the tests, correct?

**Review by chouetz — COMMENTED** [review]
Do we need a "install as much as possible" use case as well?
Regarding the install_script url, is it possible to pass a custom one (in case we don't necessarily put the script in `custom`?

**Review by chouetz — APPROVED** [review]

File: test/e2e/agentplatform.go (hunk 1 above)
Comment (pducolin) [reply test/e2e/agentplatform.go:35]: True, will change it in a follow up PR to take a param or an env var

Comment (pducolin) [reply test/e2e/agentplatform.go:35]: Actually will move it in this PR to a flag

**Comment by pducolin** [comment]
I'm closing this PR and opening a new, smaller one. Will add up based on it

**Pull request closed without merging**